docs/STRUCTURE.md

```diff
@@ -60,43 +62,21 @@ To run the updater tool:
 
 ### Database
 The database stores 3 main types of data:
-1. Release - The raw binary data extracted from Github
+1. Release - The raw binary data extracted from GitHub
 1. Stats - Download statistics. Updated at the end of a full refresh.
     - DockerStats - Broken down into each docker repository
     - GitHubStats - Broken down into each feature version
 1. Web-Cache - Cached data used to speed up requests
 
 ## Front-end
 
-The front-end is a Quarkus application that uses OpenAPI for documentation. Data is polled from the database into memory and requests are then
-serviced from that dataset.
+The front-end is a Quarkus application that uses OpenAPI for documentation.
+Data is polled from the database into memory and requests are then serviced from that dataset.
 
 ### Examples
 
-Fetch binaries and installers:
-- https://api.adoptopenjdk.net/v3/binary/latest/11/ga/mac/x64/jdk/hotspot/normal/adoptopenjdk
-- https://api.adoptopenjdk.net/v3/installer/latest/14/ea/linux/s390x/jre/openj9/large/adoptopenjdk
-
-Raw asset data:
-- https://api.adoptopenjdk.net/v3/assets/feature_releases/11/ga
-- https://api.adoptopenjdk.net/v3/assets/feature_releases/14/ea?architecture=s390x&jvm_impl=openj9
-- https://api.adoptopenjdk.net/v3/assets/latest/8/openj9
-- https://api.adoptopenjdk.net/v3/assets/version/11.0.4+11.1
-
-Release info:
-- https://api.adoptopenjdk.net/v3/info/available_releases
-- https://api.adoptopenjdk.net/v3/info/release_names
-- https://api.adoptopenjdk.net/v3/info/release_names?page_size=20&release_type=ga
-- https://api.adoptopenjdk.net/v3/info/release_versions
-
-Download statistics:
-- https://api.adoptopenjdk.net/v3/stats/downloads/total
-- https://api.adoptopenjdk.net/v3/stats/downloads/tracking
-- https://api.adoptopenjdk.net/v3/stats/downloads/tracking?source=dockerhub&feature_version=11
-- https://api.adoptopenjdk.net/v3/stats/downloads/monthly
-
-A full list of endpoints and each of the parameters can be found at https://api.adoptopenjdk.net/swagger-ui/
+A full list of endpoints and each of the parameters can be found at <https://api.adoptium.net/swagger-ui/>
 
 ### Running
-To run the front-end quarkus tool, `cd` into the `adoptopenjdk-api-v3-frontend` directory and run `../../mvnw quarkus:dev`. This will then run the tool on port 8080.
+To run the front-end quarkus tool, `cd` into the `adoptopenjdkn-api-v3-frontend` directory and run `../../mvnw quarkus:dev`. This will then run the tool on port 8080.
```

Comment:
```suggestion
To run the front-end quarkus tool, `cd` into the `adoptopenjdk-api-v3-frontend` directory and run `../../mvnw quarkus:dev`. This will then run the tool on port 8080.
```